src/Files.App/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -155,8 +142,8 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 							ShowInFtpPage = true,
 							ShowInZipPage = true,
 						},
-						new ContextMenuFlyoutItemViewModelBuilder(commands.SortAscending).Build(),
-						new ContextMenuFlyoutItemViewModelBuilder(commands.SortDescending).Build(),
+						new ContextMenuFlyoutItemViewModelBuilder(commands.SortAscending){IsToggle = true}.Build(),
+						new ContextMenuFlyoutItemViewModelBuilder(commands.SortDescending){IsToggle = true}.Build(),
```

Comment:
```suggestion
						new ContextMenuFlyoutItemViewModelBuilder(commands.SortAscending)
						{
							IsToggle = true
						}.Build(),
						new ContextMenuFlyoutItemViewModelBuilder(commands.SortDescending)
						{
							IsToggle = true
						}.Build(),
```

Reply:
There are many similar one-line codes in this file.
Should I modify all of these as well?

Reply:
I would appreciate because I have to do that in another PR🥲

Reply:
OK, I modified all similar codes in this file.

Reply:
Thank you